Cannot execute operation with "REQUEST_SELECT" type

**Version Info**
v18.1, v19

**Describe the bug**
It is not possible to start operation of type "REQUEST_SELECT" on the MySubscriptions page. After checking the service definitions, it looks like the parameter was loaded correctly.

My operation definition:
```
<Operation actionURL="http://oscm-app:8880/oscm-app/webservices/oscm-app/oscm-app/org.oscm.app.v2_0.service.AsynchronousOperationProxy?wsdl" id="SNAPSHOT">
      <LocalizedName locale="en">Create snapshot</LocalizedName>
      <LocalizedName locale="de">Snapshot anlegen</LocalizedName>
      <LocalizedDescription locale="en">Creates a snapshot.</LocalizedDescription>
      <LocalizedDescription locale="de">Legt einen snapshot an.</LocalizedDescription>
      <OperationParameter id="SERVER" mandatory="false" type="REQUEST_SELECT">
        <LocalizedName locale="en">Server</LocalizedName>
        <LocalizedName locale="de">Server</LocalizedName>
      </OperationParameter>
      <OperationParameter id="COMMENT" mandatory="true" type="INPUT_STRING">
        <LocalizedName locale="en">Comment</LocalizedName>
        <LocalizedName locale="de">Kommentar</LocalizedName>
      </OperationParameter>
    </Operation>
```
![image.png](https://images.zenhubusercontent.com/5e1c60ee5e668415ca9f76d9/2bc5570a-6149-4876-8e07-3686496b8f47)

**How to Reproduce**
Steps to reproduce the behavior:
1. Add technical service with operation which include type="REQUEST_SELECT"
2. Create service with using this technical service definition
3. Subscribe service
4. Go to MySubscriptions page
5. Choose service
6. Try to execute operation

**Observed behavior**
After selecting from the list, the website freezes, logs appear

**Additional context**
```10/09/2020_08:53:45.930 FSP_INTS-BSS: ERROR: ThreadID https-jsse-nio-8081-exec-6: ExceptionActionListener: ?? key 'ERROR' not found ??
javax.faces.FacesException: #{mySubscriptionsCtrl.operationChanged}: javax.ejb.EJBException: The bean encountered a non-application exception; nested exception is:
        javax.ejb.EJBTransactionRolledbackException: The transaction has been marked rollback only because the bean encountered a non-application exception :java.lang.NoSuchMethodError : org.oscm.types.enumtypes.OperationParameterType.isRequestValues()Z
        at com.sun.faces.application.ActionListenerImpl.processAction(ActionListenerImpl.java:118) ~[javax.faces-2.2.12.jar:2.2.12]
        at org.oscm.ui.common.ExceptionActionListener.processAction(ExceptionActionListener.java:41) [classes/:PRIVATE-BUILD-ON-6896625B496A-20201008125950]
        at org.omnifaces.eventlistener.ResetInputAjaxActionListener.processAction(ResetInputAjaxActionListener.java:197) [omnifaces-1.10.jar:1.10]
        at javax.faces.component.UICommand.broadcast(UICommand.java:315) [javax.faces-2.2.12.jar:2.2.12]
        at javax.faces.component.UIViewRoot.broadcastEvents(UIViewRoot.java:790) [javax.faces-2.2.12.jar:2.2.12]
        at javax.faces.component.UIViewRoot.processApplication(UIViewRoot.java:1282) [javax.faces-2.2.12.jar:2.2.12]
        at com.sun.faces.lifecycle.InvokeApplicationPhase.execute(InvokeApplicationPhase.java:81) [javax.faces-2.2.12.jar:2.2.12]
        at com.sun.faces.lifecycle.Phase.doPhase(Phase.java:101) [javax.faces-2.2.12.jar:2.2.12]
        at com.sun.faces.lifecycle.LifecycleImpl.execute(LifecycleImpl.java:198) [javax.faces-2.2.12.jar:2.2.12]
        at javax.faces.webapp.FacesServlet.service(FacesServlet.java:658) [javax.faces-2.2.12.jar:2.2.12]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.apache.myfaces.webapp.filter.ExtensionsFilter.doFilter(ExtensionsFilter.java:357) [tomahawk20-1.1.14.jar:1.1.14]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.oscm.ui.filter.AuthorizationFilter.handleLoggedInUser(AuthorizationFilter.java:742) [classes/:PRIVATE-BUILD-ON-6896625B496A-20201008125950]
        at org.oscm.ui.filter.AuthorizationFilter.handleProtectedUrlAndChangePwdCase(AuthorizationFilter.java:304) [classes/:PRIVATE-BUILD-ON-6896625B496A-20201008125950]
        at org.oscm.ui.filter.AuthorizationFilter.doFilter(AuthorizationFilter.java:132) [classes/:PRIVATE-BUILD-ON-6896625B496A-20201008125950]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.oscm.ui.filter.NonexistentConversationFilter.doFilter(NonexistentConversationFilter.java:37) [classes/:PRIVATE-BUILD-ON-6896625B496A-20201008125950]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.oscm.ui.filter.MarketplaceContextFilter.doFilter(MarketplaceContextFilter.java:112) [classes/:PRIVATE-BUILD-ON-6896625B496A-20201008125950]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.apache.openejb.server.httpd.EEFilter.doFilter(EEFilter.java:65) [openejb-http-7.1.1.jar:7.1.1]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.oscm.ui.filter.DisableUrlFilter.doFilter(DisableUrlFilter.java:52) [classes/:PRIVATE-BUILD-ON-6896625B496A-20201008125950]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.oscm.ui.filter.RoleBasedFilter.doFilter(RoleBasedFilter.java:82) [classes/:PRIVATE-BUILD-ON-6896625B496A-20201008125950]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.oscm.ui.filter.ClosedMarketplaceFilter.doFilter(ClosedMarketplaceFilter.java:112) [classes/:PRIVATE-BUILD-ON-6896625B496A-20201008125950]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.oscm.ui.filter.IllegalRequestParameterFilter.doFilter(IllegalRequestParameterFilter.java:81) [classes/:PRIVATE-BUILD-ON-6896625B496A-20201008125950]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.oscm.ui.filter.OidcFilter.doFilter(OidcFilter.java:129) [classes/:PRIVATE-BUILD-ON-6896625B496A-20201008125950]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.oscm.ui.filter.OidcLogoutFilter.doFilter(OidcLogoutFilter.java:88) [classes/:PRIVATE-BUILD-ON-6896625B496A-20201008125950]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.oscm.ui.filter.HttpMethodFilter.doFilter(HttpMethodFilter.java:69) [classes/:PRIVATE-BUILD-ON-6896625B496A-20201008125950]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.oscm.ui.filter.DisableUrlSessionFilter.doFilter(DisableUrlSessionFilter.java:50) [classes/:PRIVATE-BUILD-ON-6896625B496A-20201008125950]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.oscm.ui.filter.CharacterEncodingFilter.doFilter(CharacterEncodingFilter.java:57) [classes/:PRIVATE-BUILD-ON-6896625B496A-20201008125950]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) [catalina.jar:8.5.41]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) [catalina.jar:8.5.41]
        at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:199) [catalina.jar:8.5.41]
        at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96) [catalina.jar:8.5.41]
        at org.apache.tomee.catalina.OpenEJBValve.invoke(OpenEJBValve.java:44) [tomee-catalina-7.1.1.jar:7.1.1]
        at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:493) [catalina.jar:8.5.41]
        at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:137) [catalina.jar:8.5.41]
        at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81) [catalina.jar:8.5.41]
        at org.apache.tomee.catalina.OpenEJBSecurityListener$RequestCapturer.invoke(OpenEJBSecurityListener.java:97) [tomee-catalina-7.1.1.jar:7.1.1]
        at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:660) [catalina.jar:8.5.41]
        at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87) [catalina.jar:8.5.41]
        at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343) [catalina.jar:8.5.41]
        at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:798) [tomcat-coyote.jar:8.5.41]
        at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66) [tomcat-coyote.jar:8.5.41]
        at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:808) [tomcat-coyote.jar:8.5.41]
        at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1498) [tomcat-coyote.jar:8.5.41]
        at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) [tomcat-coyote.jar:8.5.41]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_262]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_262]
        at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) [tomcat-util.jar:8.5.41]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_262]
Caused by: javax.faces.el.EvaluationException: javax.ejb.EJBException: The bean encountered a non-application exception; nested exception is:
        javax.ejb.EJBTransactionRolledbackException: The transaction has been marked rollback only because the bean encountered a non-application exception :java.lang.NoSuchMethodError : org.oscm.types.enumtypes.OperationParameterType.isRequestValues()Z
        at javax.faces.component.MethodBindingMethodExpressionAdapter.invoke(MethodBindingMethodExpressionAdapter.java:101) ~[javax.faces-2.2.12.jar:2.2.12]
        at com.sun.faces.application.ActionListenerImpl.processAction(ActionListenerImpl.java:102) ~[javax.faces-2.2.12.jar:2.2.12]
        ... 74 more
Caused by: javax.ejb.EJBException: The bean encountered a non-application exception; nested exception is:
        javax.ejb.EJBTransactionRolledbackException: The transaction has been marked rollback only because the bean encountered a non-application exception :java.lang.NoSuchMethodError : org.oscm.types.enumtypes.OperationParameterType.isRequestValues()Z
        at org.apache.openejb.core.ivm.BaseEjbProxyHandler.convertException(BaseEjbProxyHandler.java:447) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.ivm.BaseEjbProxyHandler.invoke(BaseEjbProxyHandler.java:351) ~[openejb-core-7.1.1.jar:7.1.1]
        at com.sun.proxy.$Proxy431.getServiceOperationParameterValues(Unknown Source) ~[?:?]
        at org.oscm.ui.dialog.mp.usesubscriptions.MySubscriptionsCtrl.convert(MySubscriptionsCtrl.java:204) ~[classes/:PRIVATE-BUILD-ON-6896625B496A-20201008125950]
        at org.oscm.ui.dialog.mp.usesubscriptions.MySubscriptionsCtrl.operationChanged(MySubscriptionsCtrl.java:189) ~[classes/:PRIVATE-BUILD-ON-6896625B496A-20201008125950]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_262]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_262]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_262]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_262]
        at org.apache.el.parser.AstValue.invoke(AstValue.java:247) ~[jasper-el.jar:8.5.41]
        at org.apache.el.MethodExpressionImpl.invoke(MethodExpressionImpl.java:267) ~[jasper-el.jar:8.5.41]
        at org.apache.webbeans.el22.WrappedMethodExpression.invoke(WrappedMethodExpression.java:52) ~[openwebbeans-el22-1.7.6.jar:1.7.6]
        at com.sun.faces.facelets.el.TagMethodExpression.invoke(TagMethodExpression.java:105) ~[javax.faces-2.2.12.jar:2.2.12]
        at javax.faces.component.MethodBindingMethodExpressionAdapter.invoke(MethodBindingMethodExpressionAdapter.java:87) ~[javax.faces-2.2.12.jar:2.2.12]
        at com.sun.faces.application.ActionListenerImpl.processAction(ActionListenerImpl.java:102) ~[javax.faces-2.2.12.jar:2.2.12]
        ... 74 more
Caused by: javax.ejb.EJBTransactionRolledbackException: The transaction has been marked rollback only because the bean encountered a non-application exception :java.lang.NoSuchMethodError : org.oscm.types.enumtypes.OperationParameterType.isRequestValues()Z
        at org.apache.openejb.core.ivm.BaseEjbProxyHandler.convertException(BaseEjbProxyHandler.java:420) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.ivm.BaseEjbProxyHandler.invoke(BaseEjbProxyHandler.java:351) ~[openejb-core-7.1.1.jar:7.1.1]
        at com.sun.proxy.$Proxy379.getOperationParameterValues(Unknown Source) ~[?:?]
        at org.oscm.subscriptionservice.bean.ManageSubscriptionBean.getOperationParameterValues(ManageSubscriptionBean.java:135) ~[?:?]
        at org.oscm.subscriptionservice.bean.SubscriptionServiceBean.getServiceOperationParameterValues(SubscriptionServiceBean.java:5392) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_262]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_262]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_262]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_262]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext$Invocation.invoke(ReflectionInvocationContext.java:205) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext.proceed(ReflectionInvocationContext.java:186) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.oscm.interceptor.AuditLogDataInterceptor.logAuditLogData(AuditLogDataInterceptor.java:29) ~[?:?]
        at sun.reflect.GeneratedMethodAccessor592.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_262]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_262]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext$Invocation.invoke(ReflectionInvocationContext.java:205) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext.proceed(ReflectionInvocationContext.java:186) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.oscm.interceptor.ExceptionMapper.mapEJBExceptions(ExceptionMapper.java:49) ~[?:?]
        at sun.reflect.GeneratedMethodAccessor136.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_262]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_262]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext$Invocation.invoke(ReflectionInvocationContext.java:205) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext.proceed(ReflectionInvocationContext.java:186) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.oscm.interceptor.InvocationDateContainer.setTransactionTime(InvocationDateContainer.java:41) ~[?:?]
        at sun.reflect.GeneratedMethodAccessor212.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_262]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_262]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext$Invocation.invoke(ReflectionInvocationContext.java:205) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext.proceed(ReflectionInvocationContext.java:186) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.monitoring.StatsInterceptor.record(StatsInterceptor.java:191) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.monitoring.StatsInterceptor.invoke(StatsInterceptor.java:102) ~[openejb-core-7.1.1.jar:7.1.1]
        at sun.reflect.GeneratedMethodAccessor135.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_262]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_262]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext$Invocation.invoke(ReflectionInvocationContext.java:205) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext.proceed(ReflectionInvocationContext.java:186) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.interceptor.InterceptorStack.invoke(InterceptorStack.java:85) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.stateless.StatelessContainer._invoke(StatelessContainer.java:252) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.stateless.StatelessContainer.invoke(StatelessContainer.java:212) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.ivm.EjbObjectProxyHandler.synchronizedBusinessMethod(EjbObjectProxyHandler.java:265) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.ivm.EjbObjectProxyHandler.businessMethod(EjbObjectProxyHandler.java:260) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.ivm.EjbObjectProxyHandler._invoke(EjbObjectProxyHandler.java:89) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.ivm.BaseEjbProxyHandler.invoke(BaseEjbProxyHandler.java:347) ~[openejb-core-7.1.1.jar:7.1.1]
        at com.sun.proxy.$Proxy431.getServiceOperationParameterValues(Unknown Source) ~[?:?]
        at org.oscm.ui.dialog.mp.usesubscriptions.MySubscriptionsCtrl.convert(MySubscriptionsCtrl.java:204) ~[classes/:PRIVATE-BUILD-ON-6896625B496A-20201008125950]
        at org.oscm.ui.dialog.mp.usesubscriptions.MySubscriptionsCtrl.operationChanged(MySubscriptionsCtrl.java:189) ~[classes/:PRIVATE-BUILD-ON-6896625B496A-20201008125950]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_262]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_262]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_262]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_262]
        at org.apache.el.parser.AstValue.invoke(AstValue.java:247) ~[jasper-el.jar:8.5.41]
        at org.apache.el.MethodExpressionImpl.invoke(MethodExpressionImpl.java:267) ~[jasper-el.jar:8.5.41]
        at org.apache.webbeans.el22.WrappedMethodExpression.invoke(WrappedMethodExpression.java:52) ~[openwebbeans-el22-1.7.6.jar:1.7.6]
        at com.sun.faces.facelets.el.TagMethodExpression.invoke(TagMethodExpression.java:105) ~[javax.faces-2.2.12.jar:2.2.12]
        at javax.faces.component.MethodBindingMethodExpressionAdapter.invoke(MethodBindingMethodExpressionAdapter.java:87) ~[javax.faces-2.2.12.jar:2.2.12]
        at com.sun.faces.application.ActionListenerImpl.processAction(ActionListenerImpl.java:102) ~[javax.faces-2.2.12.jar:2.2.12]
        ... 74 more
Caused by: java.lang.NoSuchMethodError: org.oscm.types.enumtypes.OperationParameterType.isRequestValues()Z
        at org.oscm.domobjects.TechnicalProductOperation.isRequestParameterValuesRequired(TechnicalProductOperation.java:129) ~[?:?]
        at org.oscm.applicationservice.bean.ApplicationServiceBean.getOperationParameterValues(ApplicationServiceBean.java:918) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_262]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_262]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_262]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_262]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext$Invocation.invoke(ReflectionInvocationContext.java:205) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext.proceed(ReflectionInvocationContext.java:186) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.monitoring.StatsInterceptor.record(StatsInterceptor.java:191) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.monitoring.StatsInterceptor.invoke(StatsInterceptor.java:102) ~[openejb-core-7.1.1.jar:7.1.1]
        at sun.reflect.GeneratedMethodAccessor135.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_262]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_262]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext$Invocation.invoke(ReflectionInvocationContext.java:205) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext.proceed(ReflectionInvocationContext.java:186) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.interceptor.InterceptorStack.invoke(InterceptorStack.java:85) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.stateless.StatelessContainer._invoke(StatelessContainer.java:252) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.stateless.StatelessContainer.invoke(StatelessContainer.java:212) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.ivm.EjbObjectProxyHandler.synchronizedBusinessMethod(EjbObjectProxyHandler.java:265) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.ivm.EjbObjectProxyHandler.businessMethod(EjbObjectProxyHandler.java:260) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.ivm.EjbObjectProxyHandler._invoke(EjbObjectProxyHandler.java:89) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.ivm.BaseEjbProxyHandler.invoke(BaseEjbProxyHandler.java:347) ~[openejb-core-7.1.1.jar:7.1.1]
        at com.sun.proxy.$Proxy379.getOperationParameterValues(Unknown Source) ~[?:?]
        at org.oscm.subscriptionservice.bean.ManageSubscriptionBean.getOperationParameterValues(ManageSubscriptionBean.java:135) ~[?:?]
        at org.oscm.subscriptionservice.bean.SubscriptionServiceBean.getServiceOperationParameterValues(SubscriptionServiceBean.java:5392) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_262]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_262]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_262]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_262]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext$Invocation.invoke(ReflectionInvocationContext.java:205) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext.proceed(ReflectionInvocationContext.java:186) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.oscm.interceptor.AuditLogDataInterceptor.logAuditLogData(AuditLogDataInterceptor.java:29) ~[?:?]
        at sun.reflect.GeneratedMethodAccessor592.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_262]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_262]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext$Invocation.invoke(ReflectionInvocationContext.java:205) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext.proceed(ReflectionInvocationContext.java:186) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.oscm.interceptor.ExceptionMapper.mapEJBExceptions(ExceptionMapper.java:49) ~[?:?]
        at sun.reflect.GeneratedMethodAccessor136.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_262]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_262]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext$Invocation.invoke(ReflectionInvocationContext.java:205) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext.proceed(ReflectionInvocationContext.java:186) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.oscm.interceptor.InvocationDateContainer.setTransactionTime(InvocationDateContainer.java:41) ~[?:?]
        at sun.reflect.GeneratedMethodAccessor212.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_262]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_262]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext$Invocation.invoke(ReflectionInvocationContext.java:205) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext.proceed(ReflectionInvocationContext.java:186) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.monitoring.StatsInterceptor.record(StatsInterceptor.java:191) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.monitoring.StatsInterceptor.invoke(StatsInterceptor.java:102) ~[openejb-core-7.1.1.jar:7.1.1]
        at sun.reflect.GeneratedMethodAccessor135.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_262]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_262]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext$Invocation.invoke(ReflectionInvocationContext.java:205) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.interceptor.ReflectionInvocationContext.proceed(ReflectionInvocationContext.java:186) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.interceptor.InterceptorStack.invoke(InterceptorStack.java:85) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.stateless.StatelessContainer._invoke(StatelessContainer.java:252) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.stateless.StatelessContainer.invoke(StatelessContainer.java:212) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.ivm.EjbObjectProxyHandler.synchronizedBusinessMethod(EjbObjectProxyHandler.java:265) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.ivm.EjbObjectProxyHandler.businessMethod(EjbObjectProxyHandler.java:260) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.ivm.EjbObjectProxyHandler._invoke(EjbObjectProxyHandler.java:89) ~[openejb-core-7.1.1.jar:7.1.1]
        at org.apache.openejb.core.ivm.BaseEjbProxyHandler.invoke(BaseEjbProxyHandler.java:347) ~[openejb-core-7.1.1.jar:7.1.1]
        at com.sun.proxy.$Proxy431.getServiceOperationParameterValues(Unknown Source) ~[?:?]
        at org.oscm.ui.dialog.mp.usesubscriptions.MySubscriptionsCtrl.convert(MySubscriptionsCtrl.java:204) ~[classes/:PRIVATE-BUILD-ON-6896625B496A-20201008125950]
        at org.oscm.ui.dialog.mp.usesubscriptions.MySubscriptionsCtrl.operationChanged(MySubscriptionsCtrl.java:189) ~[classes/:PRIVATE-BUILD-ON-6896625B496A-20201008125950]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_262]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_262]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_262]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_262]
        at org.apache.el.parser.AstValue.invoke(AstValue.java:247) ~[jasper-el.jar:8.5.41]
        at org.apache.el.MethodExpressionImpl.invoke(MethodExpressionImpl.java:267) ~[jasper-el.jar:8.5.41]
        at org.apache.webbeans.el22.WrappedMethodExpression.invoke(WrappedMethodExpression.java:52) ~[openwebbeans-el22-1.7.6.jar:1.7.6]
        at com.sun.faces.facelets.el.TagMethodExpression.invoke(TagMethodExpression.java:105) ~[javax.faces-2.2.12.jar:2.2.12]
        at javax.faces.component.MethodBindingMethodExpressionAdapter.invoke(MethodBindingMethodExpressionAdapter.java:87) ~[javax.faces-2.2.12.jar:2.2.12]
        at com.sun.faces.application.ActionListenerImpl.processAction(ActionListenerImpl.java:102) ~[javax.faces-2.2.12.jar:2.2.12]
        ... 74 more```

--- a/oscm-domainobjects/javasrc/org/oscm/domobjects/TechnicalProductOperation.java
+++ b/oscm-domainobjects/javasrc/org/oscm/domobjects/TechnicalProductOperation.java
@@ -28,6 +28,7 @@ import javax.persistence.UniqueConstraint;
 
 import org.oscm.domobjects.annotations.BusinessKey;
 import org.oscm.domobjects.enums.LocalizedObjectTypes;
+import static org.oscm.types.enumtypes.OperationParameterType.REQUEST_SELECT;
 
 /**
  * @author weiser
@@ -126,7 +127,7 @@ public class TechnicalProductOperation extends
 
     public boolean isRequestParameterValuesRequired() {
         for (OperationParameter op : getParameters()) {
-            if (op.getType().isRequestValues()) {
+            if (op.getType().equals(REQUEST_SELECT)) {
                 return true;
             }
         }

